events/wmt-terminology.md

```diff
@@ -0,0 +1,55 @@
+---
+parent: Events
+title: WMT Terminology 
```

Comment:
Lowercase terminology because in our style guide title case is like that Not Like US Style